new_dawn_server/users/api/tests/test_users_api.py

```diff
@@ -181,17 +181,18 @@ def test_user_profile_get_with_height_preference(self):
                                        "height": 182,
                                    })
 
-        res = self.api_client.get("/api/v1/profile/?height__range=175,180", format="json")
+        res = self.api_client.get("/api/v1/profile/?ranking=True&viewer_id=2", format="json")
         res_data = json.loads(res.content)
-        res_data_account_arguments={
-            "birthday": "1995-01-01",
-            "phone_number": "+12345678900",
-            "gender": "M",
-            "name": "test_user2"
-        }
-        for k, v in res_data_account_arguments.items():
-            self.assertEqual(res_data['objects'][0]['account'][k], v)
-        self.assertEqual(res_data['objects'][0]['height'], 178)
+        print(res_data)
```

Comment:
Remove this. And add an assert to check that the returned results have the right length and are in a defined set?